tensorflow/lite/micro/examples/network_tester/network_tester_test.cc

```diff
@@ -70,7 +70,12 @@ TF_LITE_MICRO_TEST(TestInvoke) {
 
   tflite::MicroInterpreter interpreter(model, resolver, tensor_arena,
                                        TENSOR_ARENA_SIZE, error_reporter);
-  interpreter.AllocateTensors();
+
+  TfLiteStatus allocate_status = interpreter.AllocateTensors();
+  if (allocate_status != kTfLiteOk) {
+    error_reporter->Report("alloc failed\n");
```

Comment:
Prefer allocation or tensor allocation failed.

Reply:
Changed it to "Tensor allocation failed"